roles/kubernetes-apps/policy_controller/meta/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+dependencies:
+  - role: download
+    file: "{{ downloads.calico_policy }}"
+    when: enable_network_policy and
+      kube_network_plugin in ['calico', 'canal']
+    tags: [download, canal, policy-controller]
+  - role: policy_controller/calico
+    when: kube_network_plugin == 'calico' and
```

Comment:
two conditions seem wasteful. just change it to `when: kube_network plugin in ['canal', 'calico'] and enable_network_policy`... unless we want to enable policy even if `enable_network_policy` is false?